web/olga/analytics/utils.py

```diff
@@ -42,3 +52,18 @@ def wrapper(request, *args, **kwargs):
         return HttpResponse(status=httplib.UNAUTHORIZED)
 
     return wrapper
+
+
+def get_coordinates_by_platform_city_name(city_name):
+    """
+    Gather coordinates by platform city name with Nominatim API.
+    """
+    geo_api = requests.get('https://nominatim.openstreetmap.org/search/', params={'city': city_name, 'format': 'json'})
+
+    if geo_api.status_code == 200 and geo_api.json():
+        location = geo_api.json()[0]
```

Comment:
It would be great if you can add in a comment expected structure of the geo_api.json()